docs/how-to/add-delete-user-entity.md

```diff
@@ -2,16 +2,54 @@
 id: add-delete-user-entity
 title: How To Add and Delete The User Entity
```

Comment:
Maybe we want to call it- how to add delete authentication entity? I think that in general we need to refer to it as the authentication entity (relevant to additional places in the article), and say that by default the authentication entity is the "User" entity.
@mulygottlieb - WDYT?